tests/scanner/audit/data/cloudsql_test_rules_2.yaml

```diff
@@ -0,0 +1,8 @@
+rules:
+  - name: sample cloudsql rule to search for publicly exposed instances
+    instance_name: '*'
+    authorized_networks: '*'
+    ssl_enabled: 'False'
+    resource:
+      - type: organization
```

Comment:
Not it this one. We want a rule that raises an InvalidRulesSchemaError exception in the test